wiki/osu!stream/de.md

```diff
@@ -0,0 +1,254 @@
+- Drücken Sie und installieren Sie osu! Stream
+  - Wenn Sie aufgefordert werden, die Apple ID und das Passwort einzugeben, tun Sie dies bitte.
+- osu! stream kann abgespielt werden, nachdem der Download und die automatische Installation abgeschlossen sind.
+  - Es wird empfohlen, auch den Abschnitt "Hinzufügen von Beatmaps" zu lesen, um die neuesten kostenlosen Songs zu erhalten. Der aktuelle Download enthält nur 2 Songs.
+
+### USB-Methode
+
+** Auf dem Gerät muss [iTunes](https://www.apple.com/itunes/) installiert sein. **
+
+- Gehen Sie zu [diesem Link und laden Sie ihn herunter](http://itunes.apple.com/us/app/osu!stream/id436952197?ls=1&mt=8)
+  - Wenn Sie nach der Apple ID gefragt werden, geben Sie die Apple ID und das Passwort ein.
+- Stecken Sie das iDevice in das Gerät (das über osu! Stream verfügt).
+- Übertragen Sie die osu! -Stream-Dateien mit iTunes.
+- Der iDevice- und der osu! -Stream können abgespielt werden.
+
+## Hinzufügen von Beatmaps
+
+**Stellen Sie sicher, dass das iDevice eine Verbindung zum Internet herstellen kann (entweder über Wi-Fi oder 3G / 4G).**
+
+- Öffnen Sie den osu! -Stream.
+- Drücken Sie die osu! Symbol und es sollte zum Hauptmenü geleitet werden (oder Tutorial, wenn das erste Mal)
+- Drücken Sie die Leiste "Store"
+  - Alternativ gehen Sie zu "Play" und drücken Sie "Download more songs!"
+- Wählen Sie eine gewünschte Packung.
+  - Es wird empfohlen, zuerst Pakete herunterzuladen, die kein Geld kosten. Gratis-Packs haben anstelle des "USD $" -Kosten-Tags "FREE".
+
+## Was ist der Unterschied?
+
+- **Nur auf iOS-Geräten spielbar.** (vorerst)
+- Beatmaps werden von Grund auf neu erstellt.
+- Zum Herunterladen der Song-Packs ist kein Konto erforderlich.
+- Kann osu! Stream mit einem Twitter-Account integrieren.
+- Der Highscore kann anonym unter dem Namen "Gast" an die Site gesendet werden (falls dies vorkommt, jedoch eine Internetverbindung erforderlich ist).
+- Die Kreisuhr wird ausgetauscht und mit einer kleinen Randlinie nach unten verschoben, die je nach Genauigkeit farbig ist. \ [Bottom (Long)\]
+- **Durchschnittliches Timing** ist auf der Notenseite angegeben.
+- Kann weder alte Datensätze noch Wiederholungen speichern.
+- Skin kann nicht geändert werden und Modifikationen können (noch) nicht angewendet werden.
+- Ostereier können gefunden werden, aber es gibt keine Seite mit Erfolgen ...
+
+### Neues Spielsystem
+
+**Hinweis:** Testen Sie die neuen Gameplay-Elemente im "Tutorial" -Bildschirm.
+
+#### Finger Guides Anzeige
+
+![Finger Guides anzeigen][Os-FG.jpg]
+
+_Finger Guides display_
+
+- Der zwei durchsichtige Kreis, der im Spiel angezeigt wird, soll Anfängern helfen, zu wissen, wann sie die Objekte treffen müssen.
+  - Es zeigt die vorgeschlagene Methode, um die de design der Karte mit Perfektion.
+- Die grüne Farbe ist für die linke und die rote für die rechte Hand.
+- Diese Funktion kann aus dem Optionsmenü oder nach Abschluss des Tutorials deaktiviert werden.
+
+######Connected Beats

```

Comment:
```suggestion
#### Connected Beats
```
The heading level in the EN version is 4. Also, a space is required between the `#` and text